tailwind.config.js

```diff
@@ -1,40 +1,52 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  darkMode: ["class"],
+  darkMode: ['class'],
   content: [
     './pages/**/*.{ts,tsx}',
     './components/**/*.{ts,tsx}',
     './app/**/*.{ts,tsx}',
     './src/**/*.{ts,tsx}',
-	],
+  ],
   theme: {
     container: {
       center: true,
-      padding: "2rem",
+      padding: '2rem',
       screens: {
-        "2xl": "1400px",
+        '2xl': '1400px',
       },
     },
     extend: {
       colors: {
```

Comment:
Are the other colors actually used? Specifically 'paleBlue' and 'gradientL'? Also for white, we can just do: text-white so no need to add it.

```suggestion
     colors: {
        'primary': '#008BE7',
        'brand': '#052A42'
        grey: '#6F767B',
      },
```